charts/nutanix-csi-storage/values.yaml

```diff
@@ -38,11 +38,19 @@ imagePullPolicy: IfNotPresent
 # Storage Class settings
 #
 # choose for which mode (Volume, File, Dynamic File) storageclass need to be created
+# If you created a snapshot using this Storage Class and upgrading from 2.6.x to 3.0.0 
```

Comment:
If you have an existing VolumeSnapshot which was created using Nutanix CSI Driver < 3.0.0, you need a storageclass compatible with CSI Driver < 3.0.0 to restore a PV from that volume snapshot. Please set 
volumeClass to true for that.